test/api/test_ut_run.pks

```diff
@@ -22,6 +22,29 @@ create or replace package test_ut_run is
   procedure run_and_revalidate_specs;
   procedure generate_invalid_spec;
   procedure drop_test_package;  
+
+  procedure create_test_csl_packages;
```

Comment:
Can you make it a context and use beforeall / afterall inside context?
That way the package will get created/dropped only once.